apps/back-office/src/app/(dashboard)/clients/[id]/components/AdministrativeNoteCard.tsx

```
@@ -0,0 +1,306 @@
+                onClick={handleDelete}
+                title="Delete"
+              >
+                <Trash2 className="h-4 w-4" />
+              </Button>
+            </div>
+          )}
+        </div>
+
+        <div
+          className="text-gray-800 mt-2"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)